packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -189,14 +162,11 @@ fi
 
 installAzureFunctionsCoreTools
 
-webWorkersPackageName="web-workers"
-webApiPackageName="web-api"
-
-deployWebWorkersFunctionApp
-deployWebApiFunctionApp
+webWorkersFunctionAppName = $(deployWebAFunctionApp "web-workers-allyfuncapp" "${0%/*}/../templates/function-web-workers-app-template.json")
```

Comment:
does the `var  = value` with spaces works in bash? seems this script should fail